mitiq/calibration/tests/test_settings.py

```diff
@@ -128,17 +128,27 @@ def test_make_strategies_invalid_technique():
 def test_ZNESettings():
     circuits = ZNESettings.make_problems()
     strategies = ZNESettings.make_strategies()
-
     repr_string = repr(circuits[0])
     assert all(
         s in repr_string
         for s in ("type", "ideal_distribution", "num_qubits", "circuit_depth")
     )
-
     assert len(circuits) == 4
     assert len(strategies) == 2 * 2 * 2
 
 
+def test_PECSettings():
```

Comment:
This is a good test, but it only tests Cirq circuits.
We should test that everything works also with a different frontend. 

For example, this is what we did for ZNE:

```python
@pytest.mark.parametrize("circuit_type", SUPPORTED_PROGRAM_TYPES.keys())
def test_ZNE_workflow_multi_platform(circuit_type):
...
```

Reply:
I've added a similar test for PEC, seems to work.